package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "forever index.js",
+    "dev": "nodemon index.js",
     "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Would love to see some form of testing and/or linting to help keep the code sane. Set up [standard](https://github.com/feross/standard) maybe?